### PR TITLE
Fix typos and mention GO in analysis pipeline

### PR DESCRIPTION
**Pull request by tj8901nm**

### CODE REVIEW

File: vignettes/User-input-example.Rmd [hunk 1]
```diff
@@ -213,7 +213,9 @@ plot_activity_heatmap(activity = data_activity, signatures = unique_active_sigs)
 
 ## Active signature interpretation
 We download *Pseudomonas aeruginosa* KEGG pathway terms from the
-[TRIBE](http://tribe.greenelab.com/#/home) webserver.
+[TRIBE](http://tribe.greenelab.com/#/home) webserver. TRIBE also provides Gene
+Ontology terms. IF you want to evaluate signatures with GO terms, simply
```
Comment (gwaygenomics) [inline vignettes/User-input-example.Rmd]: capital `F` in `IF`

File: vignettes/User-input-example.Rmd [hunk 2]
```diff
@@ -213,7 +213,9 @@ plot_activity_heatmap(activity = data_activity, signatures = unique_active_sigs)
 
 ## Active signature interpretation
 We download *Pseudomonas aeruginosa* KEGG pathway terms from the
-[TRIBE](http://tribe.greenelab.com/#/home) webserver.
+[TRIBE](http://tribe.greenelab.com/#/home) webserver. TRIBE also provides Gene
```
Comment (rzelayafavila) [inline vignettes/User-input-example.Rmd]: Oh, also Tribe supports https, so the Tribe url can be changed to ``https://tribe.greenelab.com/#/home`` :smile:

Comment (rzelayafavila) [inline vignettes/User-input-example.Rmd]: Oh oops, it looks like I was a little too late!

Comment (tj8901nm) [inline vignettes/User-input-example.Rmd]: Yup, already merged, will remember this in next PR.

**Pull request merged**